docs/source/executing-operations/defer-support.mdx

```diff
@@ -204,4 +204,11 @@ The `@defer` directive is currently part of a draft-stage RFC for the GraphQL sp
 
 The Apollo Router supports the `@defer` directive as it's documented in [these edits to the RFC](https://github.com/graphql/graphql-spec/pull/742), according to the state of those edits on 2022-08-24.
 
-Entity-based `@defer` support in the Apollo Router is currently [preview functionality](/resources/product-launch-stages/#preview). We'd love your feedback on this preview! Feedback will help inform improvements to the functionality prior to its general availability. It will _also_ help inform any changes to the `@defer` RFC as it proceeds through remaining contribution stages.
+## Disabling defer
+
+Defer is enabled by default. If you wish to disable `@defer`, you can do so via router.yaml:
```

Comment:
Think it's worth adding what happens when defer is disabled to the commentary.  Does this track with your understanding?

```suggestion
Defer is enabled by default. If you wish to disable `@defer` you can do so via configuration and it will result in operations which use `@defer` being rejected by the Router:
```